src/apps/lwaftr/binding_table.lua

```diff
@@ -158,9 +155,9 @@ function BindingTable:remove_softwire_entry(entry_key_blob)
    self.softwires:remove(entry.key)
 end
 
-local lookup_key = softwire_key_t()
 function BindingTable:lookup(ipv4, port)
-   local psid = self:lookup_psid(ipv4, port)
+   local lookup_key = ffi.new(self.softwires.entry_type).key
```

Comment:
Hm, how about we just create the lookup key when we create the binding table, seems that's what we were doing in the old code. I'm thinking something like this:

```lua
local lookup_key
function BindingTable.new(psid_map, softwires)
   local ret = {
      psid_map = assert(psid_map),
      softwires = assert(softwires),
   }
   lookup_key = ret.softwires.entry_type().key
   return setmetatable(ret, {__index=BindingTable})
end
```
then this can just be reverted to the way it was and I think the code will be effectively the same, right? If I'm missing something more complicated then we can put a FIXME but I'd perfer to fix it now if it's easy.